.gitignore

```diff
@@ -0,0 +1,7 @@
+# Ignore Drupal Core Repo
```

Comment:
Is this necessary in the repo? It's ages since I did an install with this template, but doesn't the process just unzip the archive rather than create a git clone?

Reply:
From my understanding it makes a git clone https://github.com/joachim-n/drupal-core-development-project/blob/main/src/ComposerScripts.php#L32 .

Reply:
Yes, that's the git clone at /repos/drupal. My point is that the overall project isn't a git clone when you do `composer create-project`. So putting thing file there is adding a file that has no effect for people installing this as a project.

Reply:
Done, deleted both `.gitignore` and `.gitkeep`

Reply:
so `cd repos` and `git clone --branch=10.1.x https://git.drupalcode.org/project/drupal.git drupal`
are mandatory as well.